client/components/admin/AdminHistoryOodlet.vue

```diff
@@ -59,30 +56,30 @@
         return moment(this.historyOodlet.dueDate).locale('hr').format('LL');
       }
     },
-    
     methods: {
       getOffices() {
-        if(this.selectedOffice !== 'TOTALS') { return; }
-        for(let oodletId of this.historyOodlet.oodletIds) {
+        if (this.selectedOffice !== 'TOTALS') {
+          return;
+        }
+
+        this.historyOodlet.oodletIds.forEach(oodletId => {
           this.$http.get(`/oodlet/${oodletId}`)
             .then(response => {
```

Comment:
Yes you are probably right, you do get response in frontend and then you find what you need, that's the convention.